vggish_audio_segmenter.py

```diff
@@ -1,31 +1,116 @@
 __copyright__ = "Copyright (c) 2020-2021 Jina AI Limited. All rights reserved."
 __license__ = "Apache-2.0"
 
-from typing import Optional
+from typing import Optional, Iterable
 import numpy as np
 
-from jina import Executor, DocumentArray, requests
-
+from jina import Executor, Document, DocumentArray, requests
+from vggish import vggish_input, vggish_params
 
 class VGGishSegmenter(Executor):
     """
-    Encode the Document blob into embedding.
+    Segment the Document blob into audio regions.
 
     :param embedding_dim: the output dimensionality of the embedding
     """
 
-    def __init__(self, embedding_dim: int = 128, *args, **kwargs ):
+    def __init__(self, sampling_factor: int=2, chunk_duration: int = 10,
+                sample_rate: int = 44100, default_traversal_paths: Iterable[str] = ['r'], *args, **kwargs ):
         super().__init__(*args, **kwargs)
-        self._dim = embedding_dim
+        self.sampling_factor = sampling_factor # i.e. the n in sampling notation s(nT)
+        self.chunk_duration = chunk_duration
+        self.sample_rate = sample_rate
+        self.default_traversal_paths = default_traversal_paths
+
 
     @requests
-    def segment(self, docs: Optional[DocumentArray], **kwargs):
+    def segment(self, docs: Optional[DocumentArray], parameters: dict, **kwargs):
         """
-        Encode all docs with images and store the encodings in the embedding attribute of the docs.
-        :param docs: documents sent to the encoder. The docs must have `blob` of the shape `256`.
+        Encode all docs with audio and store the segmented regions in the chunks attribute of the docs.
+
+        :param docs: documents sent to the segmenter.
```

Comment:
The `docstring` should explain the usage of `parameters` and the `doc.tags`. Plus, `README.md` is missing.

Reply:
lemme add the docstring and fix a small issue of the adapted README.md on master branch